source/Calamari/Behaviors/TargetDiscoveryBehaviour.cs

```diff
@@ -224,7 +224,7 @@ public static class TargetDiscoveryHelpers
         public static ServiceMessage CreateWebAppTargetCreationServiceMessage(string resourceGroupName, string webAppName, string accountId, string role, string? workerPoolId)
         {
             var parameters = new Dictionary<string, string?> {
-                    { "name", $"{resourceGroupName}/{webAppName}" },
+                    { "name", $"azure-web-app/{resourceGroupName}/{webAppName}" },
```

Comment:
Maybe define the prefix in a const field, so that there is a single place to edit if we decide to change it again (instead of two places in the interpolated strings).

Reply:
Yeah good call, will do

Reply:
Didn't end up doing this as we were having so much trouble getting 💚 builds and we want to get this in now that we have one, will address in a later PR.

Reply:
No worries, yeah it's more important to get a new release out.